ibis/backends/base/client.py

```diff
@@ -29,6 +29,68 @@ def exists_database(self, name):
     def exists_table(self, name, database=None):
         return self.backend.exists_table(name, database)
 
+    # Non-standard methods
```

Comment:
make the comment that these are for compatiblity, ref an issue here (e.g. to remove later in time)

Reply:
ideally if you can have the same signature; the doc-string not so important

Reply:
event better you could put these in a separate mixin class e.g. ClientDispatchToBackend or ClientCompat or something and just mix here. (then its also easy for a Backend to opt *out* of compatiblity once they are good.